lib/DAC/DAC.cpp

```diff
@@ -103,13 +101,11 @@ void DAC::set_pot(uint8_t val, pot_chan channel) {
       return;
     }
   }
-  // if (val > DAC_MAX)
-  //   val = DAC_MAX;
   // setup command
   uint8_t command = get_cmd(channel);
   uint8_t oldValue = get_pot(channel);
   if (oldValue != val) {
-    //#SAVETY#: Reset constant mode on deceleration paddel touched
+    // SAFETY: Reset constant mode on deceleration paddle touched
```

Comment:
if you change #SAVETY# to SAVETY you mustdo it everywhere. My idea was to use a unique pattern for find all operations